main/FatesRestartInterfaceMod.F90

```diff
@@ -944,6 +946,17 @@ subroutine define_restart_vars(this, initialize_variables)
            units='kg/m2', veclength=num_elements, flushval = flushzero, &
            hlms='CLM:ALM', initialize=initialize_variables, ivar=ivar, index = ir_seedgerm_litt)
 
+
+    call this%RegisterCohortVector(symbol_base='fates_seed_decay', vtype=cohort_r8, &
+            long_name_base='seed bank (non-germinated)',  &
```

Comment:
Is this 'long name' accurate? Like, if it's the seed decay rate, it's not actually in the seed bank any more?

Reply:
good catch, that is not the right long name. Its a flux rate. I stole the long_name from the pool variable that was already there and forgot to update the name.  Will do.